build.zig

```diff
@@ -592,7 +592,10 @@ fn addWasiUpdateStep(b: *std.Build, version: [:0]const u8) !void {
         .cpu_arch = .wasm32,
         .os_tag = .wasi,
     };
-    target_query.cpu_features_add.addFeature(@intFromEnum(std.Target.wasm.Feature.bulk_memory));
+    // Not supported by the `wasm-opt` version in CI.
+    target_query.cpu_features_add.removeFeature(@intFromEnum(std.Target.wasm.Feature.extended_const));
+    // Not supported by `wasm2c`.
+    target_query.cpu_features_add.removeFeature(@intFromEnum(std.Target.wasm.Feature.nontrapping_fptoint));
```

Comment:
@andrewrk would it be possible to update the `wasm-opt` version on the CI machines? I guess it would also be a good opportunity to update Wasmtime (latest is 27.0.0 vs our current 10.0.2).